docker/openjdk-17-debug/Dockerfile

```diff
@@ -1,9 +1,9 @@
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
 
 ARG VERSION="dev"
 RUN apt-get update  && \
- apt-get install --no-install-recommends -q --assume-yes curl=7* wget=1.20* jq=1.6* net-tools=1.60* openjdk-17-jdk-headless=17* libjemalloc-dev=5.* && \
+ apt-get install --no-install-recommends -q --assume-yes curl=7* jq=1.6* net-tools=1.60* openjdk-17-jdk-headless=17* libjemalloc-dev=5.* && \
```

Comment:
Why did this change?

Reply:
There is no version 1.20* of wget available for Ubuntu:22.04. Unless there is a security vulnerability going to leave the default version of wget installed as part of the docker image.

Reply:
Based on the package search https://packages.ubuntu.com/search?keywords=wget updated wget to version 1.21 for the debug image